app/controllers/patients_controller.rb

```diff
@@ -20,6 +20,7 @@ def create
 
   def edit
     @note = @patient.notes.new
+    @pledge = Pledge.new(amount: @patient.pregnancy.dcaf_soft_pledge, patient: @patient, created_by: current_user, pledge_type: 'soft')
```

Comment:
Should the amount always be pulled from `dcaf_soft_pledge` and should the pledge type always be `soft`? The view currently references `dcaf_soft_pledge` as the amount